### ISSUE TITLE
 only one picture is generated each time

### ISSUE THREAD

**Issue by yangnt** [balazik/ComfyUI-PuLID-Flux#7]
Thank you for your great work!
But would you like to ask if only one picture is generated each time? If I generate two pictures, the following message will appear?

# ComfyUI Error Report
## Error Details
- **Node Type:** SamplerCustomAdvanced
- **Exception Type:** RuntimeError
- **Exception Message:** Boolean value of Tensor with more than one value is ambiguous
## Stack Trace
```
 File "C:\Users\ntzon\Projects\ComfyUI\execution.py", line 323, in execute
 output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^^^
..........
.............
## Additional Context
(Please add any additional context or steps to reproduce the error here)

**Comment by balazik** [comment]
Ah, I see by using `Repeat Latent Batch` (batch processing) I can replicate the error.
I have never tested batch processing before. I'll try to edit it to make it work, but I don't promise anything. First, I have to find out what the sampler requires for batch processing.

**Comment by balazik** [comment]
Funny thing is that I made some patches (not associated with this error) and now it automagically works. 
But also I pulled the newest ComfyUI version. So it is possible that they fixed something.

Pull the new version and the new ComfyUI version and it should work.
I tried:
- Extra options: Batch count (with randomized seed)
- EmptySD3LatentImage: batch_size
- Repeat Latent Batch: amount 

All variantion and batch generations worked for me.  If not I will reopen the issue.
